brainstormforce/ultimate-addons-for-gutenberg
# Fix: Display Condition - UAGDisplayConditions attribute was not setting for block.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/ba7d977b-a24e-4171-9869-ea50bd0803f3?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes

### Screenshots

- https://share.getcloudapp.com/KouAngzg

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Check backword Set any condition from master for the block.
- Switch to this branch and check it is working on the front as expected.
- Drag and drop the new block add display condition and check if it is working on the front or not.

### Checklist:
